app/assets/javascripts/map/templates/analysis/analysisResults.handlebars

```diff
@@ -90,16 +90,37 @@
                   <span class="stats-count" style="color: {{layer.category_color}};"><strong>{{totalAlerts}}</strong></span>
                 </li>
                 {{else}}
-                {{#if extent}}
+                  {{#if umd_as_it_happens_per}}
                   <li>
-                    <span class="stats-title">Tree cover (2000)<span class="stats-threshold" title="Click the gear icon located on the map to change the tree cover canopy density">with >{{threshold}}% <a href="#" class="canopy-button">canopy density</a></span></span>
-                    <span class="stats-count" style="color: #97bd3d;"><strong>{{extent}}</strong> ha</span>
+                    <span class="stats-title">GLAD Tree Cover Loss Alerts:</span>
+                    <span class="stats-count" style="color: {{layer.category_color}};"><strong>{{totalAlerts}}</strong></span>
                   </li>
                   {{else}}
-                  <li>
-                    <span class="stats-title">Total alerts ({{dateRange}})</span>
-                    <span class="stats-count" style="color: {{layer.category_color}};"><strong>{{totalAlerts}}</strong> {{alertsName}}</span>
-                  </li>
+                    {{#if umd_as_it_happens_cog}}
```

Comment:
We need urgently to improve this handlebars. It's horrible to maintain. We should use a helper